### PR TITLE
Feat/electron

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

We use end of support version of electron => update to `electron v20`

## Related Issue

Screenshot 2022-11-21 at 17 17 54

### CODE REVIEW

File: packages/connect-examples/electron-main-process/src/electron.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const init = () => {
         height: 775,
         webPreferences: {
             preload: path.join(__dirname, 'preload.js'),
+            sandbox: false,
```
Comment (matejkriz) [inline packages/connect-examples/electron-main-process/src/electron.js]: could you please add a comment why this is needed to be disabled?

Comment (tomasklim) [reply packages/connect-examples/electron-main-process/src/electron.js]: 1265c4fe5c1643beaf257fd15f0791a79a654cf5

Comment (tomasklim) [reply packages/connect-examples/electron-main-process/src/electron.js]: Screenshot 2022-11-22 at 13 48 46

File: shell.nix [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# the last successful build of nixpkgs-unstable as of 2022-05-31
+# the last successful build of nixpkgs-unstable as of 2022-11-21
 with import
   (builtins.fetchTarball {
-    url = "https://github.com/NixOS/nixpkgs/archive/b62ada430501de88dfbb08cea4eb98ead3a5e3e7.tar.gz";
+    url = "https://github.com/NixOS/nixpkgs/archive/19230cff31fd7562562dd25181579fa7087f0f89.tar.gz";
     sha256 = "1ppaixbncqyvy2ixskwvzggjsw20j77iy3aqyk4749dvkx0ml27f";
   })
 { };
```
Comment (tomasklim) [inline shell.nix:7]: Line 11 @vdovhanych `electron_18;`

File: packages/connect-examples/electron-main-process/src/electron.js (hunk 1 above)
Comment (matejkriz) [reply packages/connect-examples/electron-main-process/src/electron.js]: thanks

File: package.json [hunk 3]
```diff
@@ -79,7 +79,7 @@
         "type-fest": "2.12.2",
         "bcrypto": "5.4.0",
         "react": "18.2.0",
-        "electron": "18.3.5",
+        "electron": "20.3.5",
```
Comment (matejkriz) [inline package.json:82]: why not 21?

Comment (tomasklim) [reply package.json:82]: Not available in stable nix

https://search.nixos.org/packages?channel=22.05&from=0&size=50&sort=relevance&type=packages&query=Electron_21

**Review by matejkriz — APPROVED** [review]

**Comment by matejkriz** [comment]
Notes for QA: we would like to test autoupdate as soon as possible, I will prepare testing versions.

**Pull request merged**